docs/xmlprofiles.rst

```diff
@@ -614,6 +614,9 @@ List with the possible configuration parameter:
 +-----------------------------------+-----------------------------------+----------------------------------+---------+
 | Name                              | Description                       | Values                           | Default |
 +===================================+===================================+==================================+=========+
```

Comment:
I'd rather have a small table for the participant profiles tags (`<domainId>` and `<rtps>`), and then a subsection with a table for the `<rtps>` tag. The builtin section has something similar with `discovery_config`